.devcontainer/skyramp/devcontainer.json

```diff
@@ -28,6 +26,6 @@
 		35142
 	],
 	// start's the skyramp server that runs command on codespace
-	"postStartCommand": "/etc/skyramp/start.sh",
+	"postStartCommand": "mkdir -p /etc/skyramp && curl -o /etc/skyramp/start.sh https://raw.githubusercontent.com/letsramp/sample-microservices/script_branch/.devcontainer/scripts/start.sh && chmod +x /etc/skyramp/start.sh && /etc/skyramp/start.sh",
```

Comment:
I think the main thing is we'll need to move this to the test repo that Kolja made. I think it shouldn't be pulled from here. Either that or we make a independent repo for it.

Reply:
I think the Getting Started repo is great so people don't need to look at even more places if they want to fully understand what is going on